ValvePak/ValvePak/Package.Save.cs

```diff
@@ -211,22 +234,47 @@ public void Write(Stream stream)
 
 			writer.Write(NullByte);
 
-			var fileTreeSize = stream.Position - headerSize;
+			//clear sub file
+			for (ushort i = 0; i < 999; i++)
```

Comment:
Remove this loop

Reply:
This is to delete the subcontracted files produced by previous tasks. I believe that when users reduce the maximum number of bytes and recreate the subcontracted files, the existence of the previous subcontracted files can be very confusing for users

Reply:
That's up to them to clean up then, not really our job to arbitrarily loop for 1k files.

Reply:
> That's up to them to clean up then, not really our job to arbitrarily loop for 1k files. We only care that the _dir.vpk references correct chunk file which will be overwritten.

You're right, we shouldn't help users make decisions without authorization